apps/dashboard/src/lib/routing/go-and-highlight.ts

```
@@ -0,0 +1,24 @@
+  for (const [key, value] of Object.entries(routeValues)) {
+    if (value == null) continue;
+    const token = `[${key}]`;
+    if (path.includes(token)) {
+      path = path.replace(token, encodeURIComponent(value));
+    } else {
+      query.set(key, value);
+    }
+  }
+
+  query.set('highlight', sectionName);
+  goto(`${path}?${query.toString()}`);
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)